client/js/keybinds.js

```diff
@@ -127,6 +127,28 @@ for (const hotkey in colorsHotkeys) {
 	});
 }
 
+// Autocomplete bracket and quote characters like in a modern IDE
+// For example, select `text`, press `[` key, and it becomes `[text]`
+const bracketWraps = {
+	'"': '"',
+	"'": "'",
+	"{": "}",
+	"[": "]",
+	"(": ")",
+	"`": "`",
+	"~": "~",
+	"*": "*",
+	_: "_",
```

Comment:
Should `_` be surrounded by quotes for consistency?

Reply:
ESLint will say: `Unnecessarily quoted property '_' found`
(If that becomes a philosophical question about linting lol, let's merge this and improve the linting config only after)

Reply:
Time for some philosophical linting discussion!

I think we should enforce all if 1 needs it, but allow non quoted if only none of them need to be unquoted.

But yeah, I don't care about this being in this PR.